### ISSUE TITLE
[1.16.5 Rubidium 0.2.4 Oculus 1.2.2] Crash with CBMultipart and CodeChickenLib

### ISSUE THREAD

**Issue by DEWSTREAMING** [Asek3/Rubidium#51]
I am playing a Modpack called Project Architect on a server, the game crashes at certain bases but I am unsure as to what is the specific cause, I'm hoping the crash log shows something that I'm missing. The crash does not seem to take place when I am just using Rubidium 0.2.3 

I have also done "/flywheel backend off" as create mod is in this pack and I've heard that can cause crashes with shaders.

[crash-2022-04-03_16.14.56-client.txt](https://github.com/Asek3/Rubidium/files/8403925/crash-2022-04-03_16.14.56-client.txt)

As stated in the title I am on Minecraft 1.16.5 and I am using Rubidium 0.2.4 alongside with Oculus 1.2.2 I had this crash using the shaderpack ComplementaryShaders_v4.3.3 as well as Sildur's+Vibrant+Shaders+v1.32+Extreme-VL I have not tested any others

I'm quite busy but if I get some free time I will see if I can replicate the crash consistently in single player, and then try to figure out if a specific mod is to blame.

**Comment by DEWSTREAMING** [comment]
Ah, should this be remade in Oculus issues and removed from here? Made the assumption the github issues were combined like magnesium dev had theirs. My bad.

*edit* also made some progress changing which mods are in and loading a superflat world that was crashing. One set of mods caused the instant crash upon loading the world, but then splitting that set of mods into 2 different instances (all with Rubidium and oculus) neither of the two crashed at all. Upon combining them again it did crash. I'm guessing this means it may be a combination of a few mods that aren't happy when they're all together? If that's possible. Busy again but thought I'd give an update on the search

**Comment by DEWSTREAMING** [comment]
Found at least one mod causing consistent crashes from this bunch. CBMultipart and CodeChickenLib which it is dependent on I believe.
[crash-2022-04-09_19.56.10-client.txt](https://github.com/Asek3/Rubidium/files/8456664/crash-2022-04-09_19.56.10-client.txt)

I got this crash in a superflat world with CBM blocks that I placed down earlier without oculus in (rubidium was, and these blocks were invisible but not crashing). The crash happened with Rubidium and oculus in

A mod called Calcium is meant to fix some of these issues for magnesium (the invisible blocks, not sure about crashes) and although the calcium mod has added rubidium into the description, calcium currently seems to require magnesium installed in order for minecraft to even launch.
